containers/sidebar/StorageSpaceStatus.js

```diff
@@ -57,9 +57,10 @@ const StorageSpaceStatus = ({ upgradeButton }) => {
                 size="auto"
             >
                 <div className="dropDown-content">
-                    <div className="absolute top-right mt0-5 mr0-5">
-                        <button className="flex flex-items-center">
+                    <div className="absolute top-right mt0-5r mr0-5r">
+                        <button className="flex flex-items-center" title={c('Action').t`Close`}>
```

Comment:
It's important to always define `type="button"` if you are not using `<Button>` component. A `button` in a form is considered as a submit button by default.

```suggestion
                        <button type="button" className="flex flex-items-center" title={c('Action').t`Close`}>
```